.github/workflows/xtf-maven-snapshots.yaml

```diff
@@ -0,0 +1,36 @@
+name: XTF Maven Release
+
+on:
```

Comment:
Are we sure we want this? Aren't we discussing this topic in chat that we dont want tu create snapshots from all forks?

Reply:
I think it's good feature to allow to deploy snapshot from forks as it allow users to test their features in consuming test suites. This PR is just to simplify it that PGP key is not required for snapshots. The other modification will be handled in separate PR.

Reply:
As I understand this will `deploy snapshot from forks`. I am not fan of this. We can talk on Monday meeting how you mean and what other think.

Reply:
Sure, we can incorporate other changes we've discussed as well